web-ng/src/app/components/color-picker/color-picker.component.scss

```diff
@@ -12,4 +12,27 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
- */
\ No newline at end of file
+ */
+
+@import '_variables';
+
+.btn-save {
+  color: white;
+  height: 36px;
+  width: 82px;
+  border-radius: 5px;
+  font-family: 'Open Sans', Arial, sans-serif;
+  margin-left: 10px;
+}
+
+.btn-cancel {
+  height: 36px;
+  width: 82px;
+  border-radius: 5px;
+  font-family: 'Open Sans', Arial, sans-serif;
```

Comment:
FYI: Filed #604 to centralize font overrides. Ok for this PR!

Reply:
Hi @gino-m , if possible. I would love to work on the #604 issue. Thank you